scripts/helm_handle_chart.sh

```diff
@@ -56,7 +56,7 @@ if [[ -z "$KUBECONFIG_BASE64" ]]; then
                 export h="helm --kubeconfig=$BITOPS_KUBE_CONFIG_FILE"
             fi
         else
-            if [[ "${FETCH_KUBECONFIG}" == "False" ]]; then
+            if [[ "${FETCH_KUBECONFIG}" == "false" ]]; then
```

Comment:
From the conversation, this one is still coming uppercased.

The ENV var default is defined in the
https://github.com/bitops-plugins/helm/blob/ecac7a13d1c27318998d0d87d0b57cd987fb3b5c/bitops.schema.yaml#L69-L71

The context is that Helm chart is using 
https://github.com/bitops-plugins/helm/blob/ecac7a13d1c27318998d0d87d0b57cd987fb3b5c/plugin.config.yaml#L11
meaning bool var parsing is happening inside the Helm chart with some `bash`.

We'll need to ideally find the place where it's happening.

Reply:
Looks like in the `helm` pack `shyaml` tool is used for parsing the plugin schema
https://github.com/bitops-plugins/helm/blob/ecac7a13d1c27318998d0d87d0b57cd987fb3b5c/scripts/bitops-config/convert-schema.sh#L70

which probably converts bool values to the uppercase.

Worth the debugging here: 

https://github.com/bitops-plugins/helm/blob/097ad8a829ab11cb49de48c96982d27fe18ef094/scripts/bitops-config/converters/boolean.sh#L19-L20

https://github.com/bitops-plugins/helm/blob/ecac7a13d1c27318998d0d87d0b57cd987fb3b5c/scripts/bitops-config/convert-schema.sh#L129

Reply:
The alternative approach is to throw out all this insanity https://github.com/bitops-plugins/helm/blob/main/scripts/bitops-config/convert-schema.sh and switch to (now fixed) core_schema_parsing  https://github.com/bitops-plugins/helm/blob/ecac7a13d1c27318998d0d87d0b57cd987fb3b5c/plugin.config.yaml#L11